src/main/js/controls/Gantt/Gantt.js

```diff
@@ -210,7 +211,12 @@ class Gantt extends Construct {
         createAxes(this.axis, this.scale, this.config, this.svg);
         createGanttContent(this.config, this.svg);
         if (utils.notEmpty(this.config.dateline)) {
-            createDateline(this.scale, this.config, this.svg);
+            createDateline(
+                this.scale,
+                this.config,
+                this.svg,
+                settingsDictionary(this.config).transition
```

Comment:
Same rules as Graph.js apply here. You dont need a separate parameter if you are using properties within `this.config`